docs/project/2024/hadasa.jercau/index.md

```diff
@@ -0,0 +1,111 @@
+# PicoW Garden Monitor
+ A Raspberry Pi Pico project for monitoring and optimizing plant growth conditions.
+
+**Author**: JERCĂU Hadasa-Ștefana \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-hadassahj
+
+:::
+
+## Description
+
+PicoW Garden Monitor is a project designed to monitor and optimize the growth conditions of plants in gardens using the Raspberry Pi Pico microcontroller. It collects data on soil moisture, light exposure, and temperature to assess the health of plants and provide personalized care instructions for optimal growth. The project aims to include a web application too, hosted on a local server built with Rust, which would allow users to interact with the system, select plants, view real-time sensor data, and receive customized care instructions. By envisioning a locally hosted web interface as part of its development, PicoW Garden Monitor aims to offer a comprehensive solution for plant enthusiasts to cultivate healthy and thriving gardens.
+
+## Motivation
+
+The motivation behind PicoW Garden Monitor stems from a personal need to address common challenges faced in plant care. As someone who often forgets to water plants and struggles to find the optimal spot with the right amount of sunlight, there is a strong desire to find a solution that simplifies and enhances the gardening experience.
+
+By developing the PicoW Garden Monitor, the goal is to create a reliable and intuitive tool that helps overcome these challenges. With real-time monitoring of soil moisture, light exposure, and temperature, the system provides timely reminders for watering and insights into the ideal environmental conditions for plant growth. This empowers users to nurture healthy and thriving gardens, even with busy schedules or limited gardening experience.
+
+## Architecture 
+
+Main Components:
```

Comment:
Please format this, run `npm start` to see how it looks like.

Reply:
tried to solve this but i think i didnt manage to
weeee

Reply:
https://commonmark.org/help/